packaging/pom.xml

```diff
@@ -17,7 +17,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.hive</groupId>
+    <groupId>com.linkedin.hive</groupId>
     <artifactId>hive</artifactId>
     <version>1.0.1</version>
```

Comment:
Don't we want to change this to `1.0.1-avro`? Or, do you intend to do it in a separate PR?

Reply:
Planning to do in a separate PR. But can also add as a new commit in the same PR. Please let me know.

Reply:
Separate PR is fine.